### DISCUSSION TITLE
GitHub-hosted runners: Double the power for open source

### DISCUSSION THREAD

**Discussion by grandmas-favorite** [community/community#101499] [Actions]
<img width="1189" alt="Screenshot 2024-01-22 at 4 23 09 PM" src="https://github.com/community/community/assets/144172738/18b75c14-f57f-4a8f-bc61-bed2bdc4428f">

 
Hi Community citizens!

### The GitHub Actions team just posted the following update in our Product changelog:

> GitHub Actions continues its industry-leading support for the OSS community by doubling the Windows/Linux machine size to 4-vCPU runners at no cost for public repositories.

### What this means for you:
- Up to a 25% increase in performance for most CI/CD workloads without any configuration change required.
- For teams with larger workloads, the new machines offer twice the memory and over 10x the amount of storage than before.
- Simply run your public workflows using any of the existing Ubuntu or Windows [labels](https://docs.github.com/actions/using-jobs/choosing-the-runner-for-a-job#choosing-github-hosted-runners), and they now will automatically run on the new 4-core hosted runners.

If you want to read the full changelog post, you can check it out here:
- [GitHub-hosted runners: Double the power for open source](https://github.blog/2024-01-17-github-hosted-runners-double-the-power-for-open-source/)

Feel free to comment below with any questions, feedback or general thoughts!

**Comment by erickgskovik** [comment]
Hello, great news for open source!

4vCPU runner should be there for public runner.
However, I am seeing 4 vCPU GitHub in private repo too. That is wrong.

What GitHub says:
![image](https://github.com/community/community/assets/141911300/92b57303-eb04-4001-8ac2-607a11b05948)


What docker reports:
```
Server:
 Containers: 0
  Running: 0
  Paused: 0
  Stopped: 0
 Images: 14
 Server Version: 24.0.7
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: false
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 2
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
 Swarm: inactive
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: xxx
 runc version: v1.1.11-0-g4bccb38
 init version: de40ad0
 Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
 Kernel Version: 6.2.0-1018-azure
 Operating System: Ubuntu 22.04.3 LTS
 OSType: linux
 Architecture: xxx
 CPUs: 2
 Total Memory: 6.76GiB
 Name: xxx
 ID: xxx
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: githubactions
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

I would expect the information to be correct. Or the option to opt for larger runner.
